src/pages/optimized-for-change/index.zh-hans.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 为迎接改变优化
+date: '2018-12-12'
+spoiler: 什么造就了优秀的API?
+---
+
+什么造就了优秀的API？
+
+*优秀的* API 设计是便于记忆以及清晰不含糊的。它能促使代码变得可读性强，执行正确，以及性能优异, 也能帮助开发者进入[成功之坑](https://blog.codinghorror.com/falling-into-the-pit-of-success/)。
+
+我把这些设计层面叫做 “第一级”，因为这些是一个库开发者需要关注的优先事项。你可能需要在它们中的某些上做妥协和权衡，但是至少这些都是你的考虑点。
+
+然而，除非你是要将一个漫游者发往火星，不然你的代码很有可能随着时间而改变。同样，你API使用者的代码也是一样。
```

Comment:
```suggestion
然而，除非你是要将一个漫游者发往火星，不然你的代码很有可能随着时间而改变。同样，你 API 使用者的代码也是一样。
```
Add whitespaces between English and Chinese characters. See reference [here](https://github.com/sparanoid/chinese-copywriting-guidelines/blob/master/README.zh-CN.md#%E4%B8%AD%E8%8B%B1%E6%96%87%E4%B9%8B%E9%97%B4%E9%9C%80%E8%A6%81%E5%A2%9E%E5%8A%A0%E7%A9%BA%E6%A0%BC).

Reply:
Hi Chloe, thanks for reviewing! I'll talk to @boomyao to see what we can do to make this better.